PublicMapping/districtbuilder-classic
# Require email address on sign up

## Overview

This makes the email input required on account sign up.

### Checklist


### Demo

<img width="1680" alt="screen shot 2018-04-06 at 1 13 00 pm" src="https://user-images.githubusercontent.com/3959096/38434421-4445c082-399c-11e8-93b0-8b6937b3bcc4.png">

## Testing Instructions

 * Build and serve 
 * On the home page, click "Sign Up". Make sure the email field has a * on the label, the note above is says that the email is required, and that the form will not submit without entering an email.
